libraries/SITL/SIM_Precland.cpp

```diff
@@ -131,7 +131,9 @@ void SIM_Precland::update(const Location &loc, const Vector3f &position)
             static_cast<int32_t>(_origin_height),
             Location::AltFrame::ABOVE_HOME);
     Vector2f center;
-    origin_center.get_vector_xy_from_origin_NE(center);
+    if (!origin_center.get_vector_xy_from_origin_NE(center)) {
+        AP_HAL::panic("Failed to get_vector_xy_from_origin_NE");
```

Comment:
Why would this panic? Making it not healthy is probably OK, but I don't think this would be abnormal - if origin isn't set (which isn't abnormal) this method will fail.

Reply:
Changed.

Reply:
The fact that we're not failing with that fatal error probably means the test suite needs another test.